backend/competitorRoutes.py

```diff
@@ -58,9 +57,9 @@ def put(self, id):
             return {'message': 'Error: Problem Updating Competitor - Please Try Again'}, 500
 
     @requireAuthentication
-    def delete(self, id):
-        competitor = competitors.findCompetitor(id)
-        competitors.deleteCompetitor(id)
+    def delete(self, competitorId):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/brownben/munro/pullRequest?prid=4764451)